test/functional/mocks/signer.py

```diff
@@ -52,8 +52,25 @@ def displayaddress(args):
 
     return sys.stdout.write(json.dumps({"address": "bcrt1qm90ugl4d48jv8n6e5t9ln6t9zlpm5th68x4f8g"}))
 
+def signtx(args):
+    if args.fingerprint != "00000001":
+        return sys.stdout.write(json.dumps({"error": "Unexpected fingerprint", "fingerprint": args.fingerprint}))
+
+    f = open(os.path.join(os.getcwd(), "mock_psbt"), "r")
+    mock_psbt = f.read()
+    f.close()
```

Comment:
In 57b26f44b9109eb0c366adeb54bf5ebfc816b45e "rpc: send: support external signer"

Use `with`

```suggestion
    with open(os.path.join(os.getcwd(), "mock_psbt"), "r") as f:
         mock_psbt = f.read()
```